requests/token-reset.yml

```diff
@@ -0,0 +1,7 @@
+action: token_reset
```

Comment:
This should go in the admin repository. See https://github.com/conda-forge/admin-requests?tab=readme-ov-file#reset-your-feedstock-token